build/Versions.props

```diff
@@ -47,4 +35,11 @@
     <MicrosoftNETTestSdkVersion>15.0.0</MicrosoftNETTestSdkVersion>
   </PropertyGroup>
 
+  <PropertyGroup>
+    <RestoreSources>
```

Comment:
Do we need to condition these for offline build?

Reply:
No, these are overridden by the toolset when building offline. https://github.com/dotnet/roslyn-tools/blob/master/sdks/RepoToolset/tools/DefaultVersions.props#L63